test_decorators.py

```diff
@@ -0,0 +1,41 @@
+import lesson_14_
+import unittest
+
+class DecoratorsTestCase(unittest.TestCase):
+
+    def test_add_decorator(self):
+
+        sum = lesson_14_.add(2,2,2,4,9)
+        sum1 = lesson_14_.add(0,0,0)
+        sum2 = lesson_14_.add(2)
+        sum3 = lesson_14_.add(-2,2,2,4,-9)
+        sum4 = lesson_14_.add(-2,-2,-2,-4,-9)
+        sum5 = lesson_14_.add(1000,10000,0,2)
+
+        self.assertEqual(sum,19)
```

Comment:
It's better to split it into several tests.

Reply:
However... It's not nessessary to make as many tests. We only need something like this:
- test_sum_one_element;
- test_sum_several_elements.

Perhaps, that's enough. You can also add:
- test_sum_negative;
- test_sum_zeros

But I guess, I wouldn't do it for your case. :)